travis failing with gofmt issues after logviewer PR

https://travis-ci.org/kubernetes/minikube/builds/484468519?utm_medium=notification&utm_source=email

```
Checking gofmt...
diff -u cmd/minikube/cmd/config/config.go.orig cmd/minikube/cmd/config/config.go
--- cmd/minikube/cmd/config/config.go.orig	2019-01-25 17:36:28.083122940 +0000
+++ cmd/minikube/cmd/config/config.go	2019-01-25 17:36:28.083122940 +0000
@@ -222,8 +222,8 @@
 		name:        "logviewer",
 		set:         SetBool,
 		validations: []setFn{IsValidAddon},
-  },
-  {
+	},
+	{
 		name:        "gvisor",
 		set:         SetBool,
 		validations: []setFn{IsValidAddon, IsContainerdRuntime},
Gofmt errors in files: cmd/minikube/cmd/config/config.go
make: *** [test] Error 1
```

--- a/cmd/minikube/cmd/config/config.go
+++ b/cmd/minikube/cmd/config/config.go
@@ -20,9 +20,8 @@ import (
 	"encoding/json"
 	"fmt"
 	"io"
-	"strings"
-
 	"os"
+	"strings"
 
 	"github.com/spf13/cobra"
 	"k8s.io/minikube/pkg/minikube/config"
@@ -222,8 +221,8 @@ var settings = []Setting{
 		name:        "logviewer",
 		set:         SetBool,
 		validations: []setFn{IsValidAddon},
-  },
-  {
+	},
+	{
 		name:        "gvisor",
 		set:         SetBool,
 		validations: []setFn{IsValidAddon, IsContainerdRuntime},
@@ -259,7 +258,7 @@ Configurable fields: ` + "\n\n" + configurableFields(),
 }
 
 func configurableFields() string {
-	var fields []string
+	fields := []string{}
 	for _, s := range settings {
 		fields = append(fields, " * "+s.name)
 	}

--- a/cmd/minikube/cmd/config/config_view.go
+++ b/cmd/minikube/cmd/config/config_view.go
@@ -23,7 +23,6 @@ import (
 
 	"github.com/golang/glog"
 	"github.com/spf13/cobra"
-
 	"k8s.io/minikube/pkg/minikube/config"
 	"k8s.io/minikube/pkg/minikube/constants"
 )

--- a/cmd/minikube/cmd/config/configure.go
+++ b/cmd/minikube/cmd/config/configure.go
@@ -21,9 +21,8 @@ import (
 	"io/ioutil"
 	"os"
 
-	"k8s.io/minikube/pkg/minikube/service"
-
 	"github.com/spf13/cobra"
+	"k8s.io/minikube/pkg/minikube/service"
 )
 
 var addonsConfigureCmd = &cobra.Command{

--- a/cmd/minikube/cmd/config/get.go
+++ b/cmd/minikube/cmd/config/get.go
@@ -17,13 +17,12 @@ limitations under the License.
 package config
 
 import (
+	"errors"
 	"fmt"
 	"os"
 
-	"k8s.io/minikube/pkg/minikube/config"
-
-	"errors"
 	"github.com/spf13/cobra"
+	"k8s.io/minikube/pkg/minikube/config"
 )
 
 var configGetCmd = &cobra.Command{

